docs/pages/material-ui/api/accordion-summary.json

```diff
@@ -3,13 +3,7 @@
     "children": { "type": { "name": "node" } },
     "classes": { "type": { "name": "object" } },
     "expandIcon": { "type": { "name": "node" } },
-    "focusVisibleClassName": { "type": { "name": "string" } },
-    "sx": {
```

Comment:
The `sx` prop here is basically duplicated type, as it is already defined in the `ButtonBase`. We can rely on the link towards the `ButtonBase` for documentation on the sx prop.